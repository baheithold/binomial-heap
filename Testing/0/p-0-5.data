7 0 398 ; 4 17 303 ; 29 26 210 ; 29 23 409 ; 11 9 154 ; 
28 24 175 ; 9 25 88 ; 24 3 492 ; 26 15 130 ; 18 8 250 ; 
18 11 117 ; 3 10 228 ; 29 1 476 ; 20 6 202 ; 29 25 55 ; 
21 24 386 ; 14 4 38 ; 17 8 499 ; 1 6 295 ; 9 24 53 ; 
20 21 283 ; 31 25 57 ; 0 2 292 ; 6 8 385 ; 31 11 13 ; 
23 25 194 ; 28 3 419 ; 4 23 25 ; 11 2 155 ; 3 8 92 ; 
25 31 77 ; 18 9 214 ; 20 21 101 ; 28 31 153 ; 7 28 279 ; 
0 23 487 ; 4 6 450 ; 27 4 462 ; 29 19 15 ; 5 3 238 ; 
5 28 85 ; 14 10 174 ; 3 23 85 ; 2 4 441 ; 31 24 2 ; 
22 11 117 ; 28 21 230 ; 0 28 378 ; 19 7 209 ; 22 29 222 ; 
19 22 324 ; 29 9 297 ; 21 14 199 ; 25 7 26 ; 17 29 313 ; 
28 27 276 ; 18 25 110 ; 14 27 234 ; 21 24 175 ; 18 18 142 ; 
8 18 173 ; 17 23 400 ; 31 10 37 ; 7 1 186 ; 4 26 0 ; 
31 18 257 ; 24 0 150 ; 19 10 424 ; 11 21 246 ; 30 20 131 ; 
16 9 21 ; 8 17 149 ; 18 22 43 ; 19 17 490 ; 13 5 81 ; 
31 10 318 ; 31 12 169 ; 10 0 218 ; 31 2 489 ; 19 25 397 ; 
28 2 163 ; 13 20 463 ; 4 23 95 ; 21 25 462 ; 26 3 237 ; 
4 17 439 ; 17 30 342 ; 17 4 262 ; 20 30 228 ; 13 3 119 ; 
16 6 227 ; 4 25 427 ; 27 21 121 ; 20 3 345 ; 23 4 490 ; 
8 19 352 ; 7 2 244 ; 11 16 113 ; 9 8 235 ; 12 31 59 ; 
18 10 216 ; 12 18 80 ; 1 7 448 ; 4 0 215 ; 8 14 76 ; 
28 22 407 ; 30 2 232 ; 14 0 367 ; 22 15 382 ; 21 18 268 ; 
0 2 272 ; 18 3 411 ; 25 7 499 ; 25 7 280 ; 7 27 420 ; 
29 6 304 ; 31 26 76 ; 31 21 226 ; 14 28 368 ; 0 12 230 ; 
3 8 136 ; 6 23 108 ; 13 18 426 ; 20 14 211 ; 15 14 308 ; 
22 23 411 ; 16 27 372 ; 6 30 435 ; 2 14 215 ; 14 21 445 ; 
22 17 390 ; 14 13 265 ; 0 27 151 ; 15 2 348 ; 29 2 221 ; 
20 9 48 ; 16 13 238 ; 14 16 375 ; 28 19 177 ; 17 5 175 ; 
2 23 489 ; 15 12 391 ; 10 7 497 ; 12 15 102 ; 14 12 189 ; 
23 25 382 ; 4 27 135 ; 21 14 375 ; 8 7 181 ; 13 2 217 ; 
4 11 271 ; 16 11 125 ; 23 24 241 ; 7 6 191 ; 19 3 40 ; 
12 30 94 ; 8 13 118 ; 22 12 186 ; 30 25 383 ; 0 2 276 ; 
12 29 13 ; 23 27 134 ; 15 8 254 ; 22 31 378 ; 25 3 205 ; 
23 22 104 ; 4 0 477 ; 16 14 413 ; 9 18 435 ; 12 30 277 ; 
9 23 262 ; 4 21 230 ; 12 4 73 ; 12 10 492 ; 15 23 386 ; 
5 23 0 ; 5 4 215 ; 20 25 316 ; 6 5 380 ; 4 14 90 ; 
27 24 14 ; 17 14 71 ; 21 27 444 ; 31 24 372 ; 22 10 400 ; 
13 22 216 ; 18 26 413 ; 11 2 167 ; 16 26 459 ; 31 28 377 ; 
23 10 197 ; 6 2 284 ; 1 30 421 ; 25 6 66 ; 4 27 245 ; 
26 31 367 ; 20 8 37 ; 22 23 83 ; 17 26 66 ; 13 3 479 ; 
24 16 246 ; 26 25 129 ; 24 18 211 ; 30 1 417 ; 
