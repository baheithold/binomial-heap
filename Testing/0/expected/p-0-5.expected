0: 1
1: 0(1)84 7(1)32
2: 2(7)78
3: 3(2)143 6(2)241
4: 5(3)59
5: 4(5)38
weight: 675
